src/omv/imlib/framebuffer.h

```diff
@@ -34,6 +34,8 @@ typedef struct framebuffer {
     volatile int32_t tail;
     bool check_head;
     int32_t sampled_head;
+    // Disable autoflush policy on full to prevent stale frames.
+    bool disable_full_flush;
```

Comment:
```suggestion
    bool disable_flush;
```
This should be in sensor_t.

Reply:
It's really a frame buffer policy option.

But okay. Note, the whole frame buffer struct and sensor struct could be one.